app/service/data_svc.py

```diff
@@ -298,21 +302,11 @@ async def _verify_abilities(self):
                               self.get_service('file_svc').special_payloads if special_payload.startswith('.')]
         cleanup_abilities = await self.locate('abilities', dict(ability_id='4cd4eb44-29a7-4259-91ae-e457b283a880'))
         for ability in await self.locate('abilities'):
-            if not ability.name:
-                ability.name = '(auto-generated)'
-                self.log.warning('Missing name for ability: %s' % ability.ability_id)
-            if not ability.description:
-                ability.description = '(auto-generated)'
-                self.log.warning('Missing description for ability: %s' % ability.ability_id)
-            if not ability.tactic:
-                ability.tactic = '(auto-generated)'
-                self.log.warning('Missing tactic for ability: %s' % ability.ability_id)
-            if not ability.technique_id:
-                ability.technique_id = '(auto-generated)'
-                self.log.warning('Missing technique ID for ability: %s' % ability.ability_id)
-            if not ability.technique_name:
-                ability.technique_name = '(auto-generated)'
-                self.log.warning('Missing technique name for ability: %s' % ability.ability_id)
+            required_fields = ['name', 'description', 'tactic', 'technique_id', 'technique_name']
```

Comment:
nice

Reply:
Moved the `required_fields` definition up.

Reply:
yea better place